src/stdfblib/ita/debug_device/DebugDevice.h

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Jose Cabral
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Jose Cabral - initial implementation
+ *******************************************************************************/
+
+#pragma once
+
+#include "RMT_DEV.h"
+#include "DebugMGR.h"
+
+/**
+ * @brief Device that adds debug commands to the device. The commands are defined in DebugMGR
+ * 
+ */
+class DebugDevice : public RMT_DEV {
+public:
+
+  DebugDevice(const std::string &paMGRID = "localhost:61499");
```

Comment:
shouldn't this be the default opc ua setup?

Reply:
I double check the OPCUA_DEV and it seems I changed it wronly in my last commit. It seems this value gets passed to the data input of the RMT_DEV which ultimately is being used by the RMT_RES as incoming port. So, I seem this is correct, and the one on OPCUA_DEV is now wrong. 

The `opc.tcp://localhsot:4840` is being set in the opcua local handler, but this parameter is not passed to it in any form

Reply:
I'm not sure if we have here a general design issue.

Reply:
> I'm not sure if we have here a general design issue. Especially under consideration of my comment above.

Reply:
I think the `paMGRID` was always the only possible configuration for a forte device. One idea I can think of is to have a `DeviceConfig` class which is passed as a parameter and the device stored it and offers it in its interface as a readonly, so it's not changed after creation.

This `DeviceConfig` class could be a simple dictionary<string, string> and needed config (such as `paMGRID` or the endpoint needed by the opcua server) can be retrieved by the class that needs it.

Reply:
For my your proposal looks like an overkill. The MGR_ID was always some configuration parameter that is used in the 4diac IDE to tell 4diac IDE how to reach the device and on 4diac FORTE on how to setup any listening communication system so that 4diac IDE can connection and send management commands. What speaks against the fact that we use a device specific default MGR_ID and allow the users to specify a specific one via a command line parameter. With that 90% of the existing code would already work and we just have to correct the OPC UA device and maybe clear up some commandline parsing. But maybe that is better discussed next week in person at the 4days of Eclipse 4diac Winter School.

Reply:
loved the last sentence!

Reply:
but I changed it back in the opcua dev again as it was before so the normal MGR still works in parallel to the opcua one